### ISSUE TITLE
svgexport produces huge file size while converting from SVG to PNG

### ISSUE THREAD

**Issue by markvital** [shakiba/svgexport#60]
I'm converting [this icon](https://user-images.githubusercontent.com/742796/42316816-4f6491c8-807d-11e8-9263-688d630117c8.png) from SVG to PNG in 128x128 resolution

I am using this command for conversion:

```
svgexport woman-test.svg woman-test.png :128
```

Here is what I've got:
![woman-test](https://user-images.githubusercontent.com/742796/42316961-b873eb6e-807d-11e8-8b63-f77d3420b0b9.png)

The result file is much bigger then the one Illustrator produces (via 'save for web') **66 KB** vs **983 bytes** in Illustrator.

Here is the version saved in Illustrator
![woman in-ai](https://user-images.githubusercontent.com/742796/42316924-9c99bb76-807d-11e8-8bdb-3efe874b3b29.png)

**Comment by mingrammer** [comment]
I got same problem

**Comment by markvital** [comment]
@mingrammer it looks like the repo is dead :(

**Comment by mingrammer** [comment]
Yes it does.. we should find an another one or make a similar tool ourselves.

**Comment by MichaelHeerklotz** [comment]
Maybe the puppeteer version works better in this case https://github.com/MichaelHeerklotz/svgexport

**Comment by shakiba** [comment]
This is fixed in +0.4 thanks to great work by @MichaelHeerklotz!

**Comment by markvital** [comment]
@shakiba @MichaelHeerklotz Great, thank you!

I did some testing and now the file size is much closer to the Illustrator.

Here are the results with the same icon I posted before:
![Screen Shot 2020-04-09 at 5 21 49 PM](https://user-images.githubusercontent.com/742796/78951673-ca370b00-7a87-11ea-838d-aade45cb4205.png)

From left to right: SVG, Adobe Illustrator Export (via Save for Web), svgexport, svgexport optimized with [ImageOptim](https://imageoptim.com/mac)

I think the results could be even better, if there is an option to set up 8-bit color profile for PNG export, (PNG-8).
